### DISCUSSION TITLE
Add number of dependents to crate metadata

### DISCUSSION THREAD

**Discussion by amab8901** [rust-lang/crates.io#8864] [Feature Requests]
When I browse crates.io, I wanna discover new crates that are likely to have valuable use-cases in my project (such as itertools, tokio, serde, cargo-chef, smallvec, etc). The best way I know to achieve this right now, is to "Browse All Crates" and look at "Recent Downloads" or "All-time Downloads", interpreting these as metrics of popularity. But I realized that popularity can be more accurately measured by the number of dependents.

# Example:
https://crates.io/crates/thiserror-impl
this crate has 200 million all-time downloads and 31 million recent downloads. By these metrics, one might think "wow this is heavily used by Rust developers, so I should probably have a look at it since it may come in handy in my own projects!". But when we look at the number of dependents (https://crates.io/crates/thiserror-impl/reverse_dependencies), there's only 6 of them. Having this information available right in the search results would let me more quickly/conveniently identify which crates are likely to be relevant, without having to open each crate and click "dependents". Having the number of dependents specified will also pave the way for implementing a sorting of all crates based on the number of dependents, which is a much more meaningful metric of popularity than the ones currently used.

**Comment by Turbo87** [comment]
this seems like a duplicate of https://github.com/rust-lang/crates.io/discussions/8741...
